data/compare.py

```diff
@@ -0,0 +1,58 @@
+import nlu
+
+from questions import ExampleQuestion
+
+
+class Intent(object):
+    def __init__(self, intent: str, score: float):
+        self.intent = intent
+        self.score = score
+
+    def __repr__(self):
+        return '<Intent {intent} {score}>'.format(
+            intent=self.intent, score=self.score)
+
+
+class QueryResult(object):
```

Comment:
What kind of query? What is the result?

Reply:
It's a JSON object of the result more like the DTO.

```json
{
   "query": "Book me a flight to Cairo",
   "topScoringIntent": {
       "intent": "BookFlight",
       "score": 0.9887482
   },
   "intents": [
       {
           "intent": "BookFlight",
           "score": 0.9887482
       },
       {
           "intent": "None",
           "score": 0.04272597
       },
       {
           "intent": "LocationFinder",
           "score": 0.0125702191
       },
       {
           "intent": "Reminder",
           "score": 0.00375502417
       },
       {
           "intent": "FoodOrder",
           "score": 3.765154E-07
       },
   ]
}
```

Reply:
might be renamed to `SimilarityResult` would be better.